vsintegration/tests/FSharp.Editor.Tests/BraceMatchingServiceTests.fs

```diff
@@ -92,6 +102,35 @@ type BraceMatchingServiceTests() =
     member this.BraceInInterpolatedStringSimple() =
         this.VerifyBraceMatch("let x = $\"abc{1}def\"", "{1", "}def")
 
+    [<Fact>]
+    member this.BraceInInterpolatedStringWith3Dollars() =
+        this.VerifyBraceMatch("let x = $$$\"\"\"abc{{{1}}}def\"\"\"", "{{{", "}}}")
```

Comment:
It would be helpful to explain why 3$ are used here - or add tests for 2$.

Reply:
Added s'more tests

Reply:
Awesome